examples/vmwithcontracts/actions/publish.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
```

Comment:
Can we add unit tests for the newly added actions?

Reply:
Will create an issue for this